webapp/loggedin/appErrors/reducer.js

```diff
@@ -0,0 +1,19 @@
+import { exportReducer } from '../../utils/reduxUtils'
+
+import {
+  appErrorCreate,
+  appErrorDelete,
+  systemErrorThrow,
+} from './actions'
+
+import * as AppState from '../../app/appState'
+
+const actionHandlers = {
+  [appErrorCreate]: (state, { error }) => AppState.assocAppError(error)(state),
+
+  [appErrorDelete]: (state, { error }) => AppState.dissocAppError(error)(state),
+
+  [systemErrorThrow]: (state, { error }) => AppState.assocSystemError(error)(state),
+}
+
```

Comment:
``` AppState.assocAppError```, ```AppState.dissocAppError``` and ```AppState.assocSystemError``` should be moved to the new module AppErrorsState

Reply:
If `assocSystemError` should be in AppErrorsState, should `throwSystemError` really be in `app/actions`?

Reply:
these are two different types of errors. systemError just locks the application and it's used by appRouterSwitch (which is part of the main bundle) - app errors can happen in an rest request, and it's included in the second bundle. that's why i would prefer to separate them.
that's also the reason of this refactor. all code in loggedin folder should be isolated from the main bundle as much as possible (see appViewExport).
what do you think?